backend/app/management/commands/syncdb.py

```diff
@@ -2,9 +2,7 @@
 Django management command syncdb
 
 Syncs local db with data from project Google Sheet
-
-TODO(ra): link Google Sheet here
-"""
+s"""
```

Comment:
Not blocking a merge, but just noting this errant `s` -- I know your next work is going to involve this file, so cleanup then.